maven-projects/spark/snb-graphar-bridge/src/main/scala/org/apache/graphar/datasources/ldbc/bridge/LdbcGraphArBridge.scala

```diff
@@ -0,0 +1,1100 @@
+        tempStaticOutputPath,
+        spark,
+        s"${graph_name}_static",
+        vertex_chunk_size,
+        edge_chunk_size,
+        file_type
+      )
+
+      // 2. Merge directory structures
+      mergeDirectoryStructures(
+        tempStaticOutputPath,
+        tempDir.streamingPath,
+        output_path
+      )
+
+      // 3. Generate unified standard YAML metadata
+      generateHybridStandardYamls(
+        output_path,
+        dataCollector,
+        graph_name,
+        vertex_chunk_size,
+        edge_chunk_size,
+        file_type
+      )
+
+      OutputExecutionResult(
+        success = true,
+        strategy = OutputStrategy.HYBRID_DOCUMENTED,
+        outputFormat = "hybrid_with_standard_metadata",
+        totalEntities = dataCollector.getTotalEntityCount()
+      )
+
+    } catch {
+      case e: Exception =>
+        logger.error("Hybrid documented strategy execution failed", e)
+        OutputExecutionResult(
+          success = false,
+          strategy = OutputStrategy.HYBRID_DOCUMENTED,
+          outputFormat = "failed",
+          totalEntities = 0,
+          error = Some(e.getMessage)
+        )
+    }
+  }
+
+  // Helper method implementations
+  private def getScaleFactor(spark: SparkSession): Double = {
+    // Get scale factor from Spark configuration or system properties
+    spark.conf
+      .getOption("ldbc.scale.factor")
+      .map(_.toDouble)
+      .getOrElse(0.003) // Default scale factor
+  }
+
+  private def createTempDirectories(basePath: String): TempDirectoryInfo = {
+    val tempDir = TempDirectoryInfo(
+      basePath = basePath,
+      staticPath = s"$basePath/.temp_static",
+      streamingPath = s"$basePath/.temp_streaming"
+    )
+
+    Files.createDirectories(Paths.get(tempDir.staticPath))
+    Files.createDirectories(Paths.get(tempDir.streamingPath))
+
+    tempDir
+  }
+
+  private def cleanupTempDirectories(tempDir: TempDirectoryInfo): Unit = {
+    try {
+      if (Files.exists(Paths.get(tempDir.staticPath))) {
+        deleteDirectory(new File(tempDir.staticPath))
+      }
+      if (Files.exists(Paths.get(tempDir.streamingPath))) {
+        deleteDirectory(new File(tempDir.streamingPath))
+      }
+    } catch {
+      case e: Exception =>
+        logger.warn("Failed to cleanup temporary directories", e)
+    }
+  }
+
+  private def deleteDirectory(directory: File): Boolean = {
+    if (directory.exists()) {
+      directory.listFiles().foreach { file =>
+        if (file.isDirectory) {
+          deleteDirectory(file)
+        } else {
+          file.delete()
+        }
+      }
+      directory.delete()
+    } else {
+      true
+    }
+  }
+
+  /**
+   * Scan temporary streaming directory to discover generated dynamic entities
+   * Returns Map[EntityType, (Path, IsVertex)]
+   */
+  private def discoverStreamingEntities(
+      tempStreamingPath: String,
+      file_type: String
+  ): Map[String, (String, Boolean)] = {
+    import java.nio.file.{Files, Paths}
```

Comment:
I would prefer to use `org.apache.hadoop.fs.{FileSystem, Path}` (https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) instead. I can easily imagine the case we want to write data to distributed storage like S3 or HDFS instead of local disks.